utils.go

```diff
@@ -104,11 +104,16 @@ func GetLSM() (lsm string, err error) {
 func getPlatformVersion(output, platform string) string {
 	flagRe := regexp.MustCompile("version_id" + `=([^ \n]*)`)
 	vals := flagRe.FindStringSubmatch(output)
-
+	if vals == nil {
+		flagRe := regexp.MustCompile("version" + `=([^ \n]*)`)
+		vals = flagRe.FindStringSubmatch(output)
```

Comment:
I think you could match both options with one regex, something like `version[_id]*=([^ \n]*)` (within backtick quotes). Try it out at regex101.com to check.

Reply:
Less is better :) 
would make the change :)